index.js

```diff
@@ -27,7 +39,70 @@ app.use(express.json());
 // IDEAS METHODS
 
 app.get('/ideas', (req, res) => {
-    res.status(200).json(getIdeas());
+    let sortedIdeasList;
+    if (req.query.sortBy === 'all') {
+        sortedIdeasList = getIdeas();
+    }
+    if (req.query.sortBy === 'totalDonationSum') {
+        sortedIdeasList = sortedByDonationSumIdeas();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'accepted') {
+        sortedIdeasList = approvedIdeasList();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'pending') {
+        sortedIdeasList = pendingIdeasList();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'rejected') {
+        sortedIdeasList = rejectedIdeasList();
+    }
+    try {
+        res.json(sortedIdeasList);
+        res.status(200);
```

Comment:
Can you chain `res.json` and `res.status`? Instead of having them as two separate expressions.

Reply:
Updated. Tested. Will be commited with another issue.